src/types.rs

```diff
@@ -54,6 +54,35 @@ impl BERDecodable for RsaPkcs15<Sha256> {
     }
 }
 
+pub struct EcdsaPkcs15<H>(pub H);
```

Comment:
This struct name makes no sense

Reply:
This follows same convention as: (and whole code follows the Rsa example)

```
pub struct RsaPkcs15<H>(pub H);
```

I can change it as needed.

Reply:
It's not a convention. RSA is defined in PKCS 1.5, ECDSA is not.